lib/internal/bootstrap_node.js

```diff
@@ -54,6 +54,8 @@
 
     _process.setupRawDebug();
 
+    NativeModule.require('internal/url');
```

Comment:
You could add a comment that mentions the side effect you’re using this for, e.g.
```js
// Make sure setURLConstructor() is called before the native
// URL::ToObject() method is used.
NativeModule.require('internal/url');
```